gha/test/action.yaml

```diff
@@ -37,14 +38,15 @@ runs:
         fi
         echo "::set-output name=pack-name::${PACK_NAME}"
 
-    - name: 
+    - name:
+      shell: bash
       env:
         VIRTUALENV_DIR: "~/virtualenv"
         FORCE_CHECK_ALL_FILES: true if on default branch
         ROOT_DIR: pack directory
-        CI_DIR: ${{ github.action_path + "/../.." }}
+        CI_DIR: ${{ format('{0}{1}', github.action_path, '/../..') }}
```

Comment:
Here is where CI_DIR is populated - I kind of guessed on the relative path though, so feel free to adjust this if it's not right.

Reply:
![single-vs-double](https://user-images.githubusercontent.com/92544319/143074431-381cb35b-742e-49da-9044-a39ee028215d.png)